### ISSUE TITLE
ERROR:Invalid UI version in the JSON-RPC response

### ISSUE THREAD

**Issue by seawindh** [storj/gateway-st#75]
### What version of Gateway-ST are you using (`gateway version`)?

<pre>
$ gateway version
Development build

</pre>
**I tried the lasted version and  [v1.77.3](https://github.com/storj/storj/releases/tag/v1.77.3) , and get the same results.
Thanks a lot!!!**

### What operating system and processor architecture are you using (`uname -mrsv`)?

<details><summary><code>uname -mrsv</code> Output</summary><br><pre>
$ uname -mrsv
Linux 5.19.0-41-generic #42~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Tue Apr 18 17:40:00 UTC 2 x86_64
</pre></details>

### What did you do?
I have followed all steps of [Test network · storj/storj Wiki · GitHub 2](https://github.com/storj/storj/wiki/Test-network) but at the time of login through http://localhost:11000/minio/login I got the Error : “Invalid UI version in the JSON-RPC response”.
![image](https://user-images.githubusercontent.com/24694940/235856967-cc60efbd-6973-4dcd-a7b3-cf75b2bf44a9.png)

and I try the follow command and get the same result.

 
```
gateway run \
--access "$(storj-sim network env GATEWAY_0_ACCESS)" \
--minio.access-key "$(storj-sim network env GATEWAY_0_ACCESS_KEY)" \
--minio.secret-key "$(storj-sim network env GATEWAY_0_SECRET_KEY)" \
--server.address 127.0.0.1:7777

```
![image](https://user-images.githubusercontent.com/24694940/235857042-a11ba780-78ad-4bdc-94e3-acb3b985d0ad.png)

**Comment by amwolff** [comment]
Thanks for the report. While we don't have a fix for this behavior yet, please take a look at #65, where an immediate fix is discussed.

**Comment by amwolff** [comment]
Closing this as we released [v1.9.0](https://github.com/storj/gateway-st/releases/tag/v1.9.0) containing a fix for this issue.
